### DISCUSSION TITLE
Running asreview docker contain behind reverse (nginx) proxy on subpath

### DISCUSSION THREAD

**Discussion by eriknl1982** [asreview/asreview#1667] [Q&A]
I am attempting to deploy the ASReview Docker container in a environment and need to access it via a sub-path on my domain, specifically https://mydomain.com/asreview. To achieve this, I've set up a reverse proxy using Nginx with the following configuration:

```
location /asreview {

    rewrite ^/asreview/(.*)$ /$1 break;

    proxy_pass http://asreview:5000/;
    proxy_redirect default;
    proxy_redirect http://asreview:5000/ /asreview/;

```
This setup initially seems to work, as the ASReview application's landing page loads correctly. However, I encounter issues with the static files (CSS, JS, etc.). The application requests these files from the root path (https://mydomain.com/static/...) instead of the intended sub-path (https://mydomain.com/asreview/static/...). This results in broken styles and scripts because those resources do not exist at the root path.

Examples of the incorrect URLs being requested:

https://mydomain.com/static/css/main.9e747be3.css
https://mydomain.com/static/js/main.7e363081.js
I'm looking for a way to adjust either the Nginx configuration or the ASReview application setup so that it correctly references static files when accessed through a sub-path. Has anyone managed to successfully deploy ASReview in a similar setup or encountered and resolved this issue?

**Comment by cskaandorp** [comment]
Hey Erik,

Let me start by saying that your question is way out of my comfort zone. I have setup ASReview on several servers now with Docker but never have attempted to do it like that. I have used subdomains and never ran into trouble. Maybe you are really set on running ASReview via a sub-path, but if not, have you tried it with a subdomain like asreview.mydomain.com??

Cspr
